react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton(props) {
+  return <button>Click Me!</button>;
+}
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+const element = CustomButton();
+root.render(element);
```

Comment:
We don't call components like regular functions, we use them as JSX:

```JSX
root.render(<CustomButton />);
```

Reply:
Oh ok, I'll go ahead and fix that.  Thank you

Reply:
<img width="227" alt="Screen Shot 2022-05-18 at 1 04 45 PM" src="https://user-images.githubusercontent.com/20695764/169146525-22f6d9d6-cbd6-418c-acde-8a7b48493407.png">